engine/src/main/java/org/terasology/config/AudioConfig.java

```diff
@@ -20,49 +20,89 @@
 import java.beans.PropertyChangeSupport;
 
 /**
+ * For configuring the audio of the game
  */
 public class AudioConfig {
     public static final String SOUND_VOLUME = "soundVolume";
     public static final String MUSIC_VOLUME = "musicVolume";
 
-    private float soundVolume;
-    private float musicVolume;
-    private boolean disableSound;
+    private float soundVolume;          // The sound volume
+    private float musicVolume;          // The music volume
+    private boolean disableSound;       // The boolean property of whether sound is disabled or not
 
     private transient PropertyChangeSupport propertyChangeSupport = new PropertyChangeSupport(this);
-
+    /**
+     * Returns the current sound volume
+     *
+     * @return a float representing the current sound volume
+     */
```

Comment:
Add new information if you document a method.
the method signature already explains, that this returns the sound volume, also that this will be a float.
What would be interesting to know, is the value range of the float (yes, this requires code-digging ;) ).